### PR TITLE
General documentation improvements

### PR DESCRIPTION
**Pull request by michel-tricot**
## What
General documentation improvements

There is an issue with gitbook that makes it create new assets and update the files where they are used. I asked them why but still no answer.

### CODE REVIEW

File: docs/contributing/contributing-to-airbyte/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ description: 'We love contributions to Airbyte, big or small.'
 
 # Contributing to Airbyte
 
-We hope you are enjoying Airbyte and look forward to hearing your feedback. Whether you want to raise a bug report, a feature/integration request or just request a correction to something, we really value our community input.
+We hope you are enjoying Airbyte and we look forward to hearing your feedback. Whether you want to raise a bug report, a feature/integration request or just request a correction to something, we are eager to get  our community input.
```
Comment (sherifnada) [inline docs/contributing/contributing-to-airbyte/README.md:7]:
```suggestion
We hope you are enjoying Airbyte and we look forward to hearing your feedback. Whether you want to raise a bug report, a feature/integration request or just request a correction to something, we are eager to get input from our community.
```

File: docs/contributing/contributing-to-airbyte/README.md [hunk 2]
```diff
@@ -34,9 +34,9 @@ You are welcome to add your own reactions to the existing issues. We will take t
 
 ## **Requesting new Features**
 
-You are very welcome to create new issues on this project to request new features.
+You are welcome to create new issues on this project to request new features.
```
Comment (sherifnada) [inline docs/contributing/contributing-to-airbyte/README.md:37]: "You are welcome ..." makes it sound non-conclusive. How about: 

```suggestion
To request new features, please create an issue on this project.
```

File: docs/contributing/contributing-to-airbyte/developing-locally.md [hunk 3]
```diff
@@ -1,36 +1,41 @@
 # Developing Locally
 
-## Build with `gradle`
-
 Airbyte uses `java 14` , `node 14` and `Docker`
 
+To start contributing:
+
+1. Start by [forking](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo) the repository
+2. Clone the fork on your workstation:
```
Comment (sherifnada) [inline docs/contributing/contributing-to-airbyte/developing-locally.md:8]:
```suggestion
1. Clone the fork on your workstation:
```

File: docs/contributing/contributing-to-airbyte/developing-locally.md [hunk 4]
```diff
@@ -1,36 +1,41 @@
...
+
+   ```bash
+   git clone git@github.com:{YOUR_USERNAME}/airbyte.git
+   cd airbyte
+   ```
+
+3. You're ready to start!
```
Comment (sherifnada) [inline docs/contributing/contributing-to-airbyte/developing-locally.md:15]:
```suggestion
1. You're ready to start!
```

File: docs/contributing/contributing-to-airbyte/developing-locally.md [hunk 5]
```diff
@@ -44,14 +49,11 @@ VERSION=dev docker-compose up
 
 ## Develop on individual applications
 
+The easiest way to work one of Airbyte's module is to spin up the whole Airbyte system on your workstation, and shutdown the module you want to work on.
```
Comment (sherifnada) [inline docs/contributing/contributing-to-airbyte/developing-locally.md:52]:
```suggestion
The easiest way to develop on one of Airbyte's modules is to spin up the whole Airbyte system on your workstation, and shutdown the module you want to work on.
```

File: docs/contributing/contributing-to-airbyte/updating-documentation.md [hunk 6]
```diff
@@ -2,29 +2,27 @@
 
 Our documentation uses [GitBook](https://gitbook.com) and all the [Markdown](https://guides.github.com/features/mastering-markdown/) files are stored in our Github repository.
 
-There are 3 different ways you can update the documentation.
+## Modify on GitHub
 
-## `Edit on GitHub` link
+1. Start by [forking](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo) the repository.
+2. Clone the fork on your workstation:
```
Comment (sherifnada) [inline docs/contributing/contributing-to-airbyte/updating-documentation.md:8]:
```suggestion
1. Clone the fork on your workstation:
```

File: docs/contributing/contributing-to-airbyte/updating-documentation.md [hunk 7]
```diff
@@ -2,29 +2,27 @@
...
-That will get you directly to the file on GitHub. You can use GitHub's editor to update the file. Once you're satisfied with your change you can submit a PR.
-
-We recommend that method for changes limited to one file \(typos, minor updates\).
-
-## Modify the repository
-
-```bash
-$ git clone git@github.com:airbytehq/airbyte.git
-```
+3. Modify the documentation.
```
Comment (sherifnada) [inline docs/contributing/contributing-to-airbyte/updating-documentation.md:15]:
```suggestion
1. Modify the documentation.
```

File: docs/contributing/contributing-to-airbyte/updating-documentation/README.md [hunk 8]
```diff
@@ -2,29 +2,27 @@
 
 Our documentation uses [GitBook](https://gitbook.com) and all the [Markdown](https://guides.github.com/features/mastering-markdown/) files are stored in our Github repository.
 
-There are 3 different ways you can update the documentation.
+## Modify on GitHub
```
Comment (sherifnada) [inline docs/contributing/contributing-to-airbyte/updating-documentation/README.md:5]: Seems like there are identical files `README.md` as well as a `updating-documentation.md` -- comments apply to both :)

**Review by sherifnada — APPROVED** [review]
Awesome!

File: docs/contributing/contributing-to-airbyte/updating-documentation/README.md (hunk 8 above)
Comment (michel-tricot) [reply docs/contributing/contributing-to-airbyte/updating-documentation/README.md:5]: weird
I'll see how to fix it

File: docs/SUMMARY.md [hunk 9]
```diff
@@ -23,15 +23,15 @@
...
-    * [Source Documentation Template](contributing/contributing-to-airbyte/updating-documentation/source-documentation-template.md)
+  * [Updating Documentation](contributing/contributing-to-airbyte/updating-documentation.md)
+  * [Documentation Templates](contributing/contributing-to-airbyte/documentation-templates/README.md)
+    * [Source Template](contributing/contributing-to-airbyte/documentation-templates/source-template.md)
 * [Code of Conduct](contributing/code-of-conduct.md)
 
 ## Architecture
 
 * [High Level Overview](architecture/high-level-overview.md)
-* [Configuration Data Model](architecture/configuration-data-model.md)
```
Comment (michel-tricot) [inline docs/SUMMARY.md:33]: @cgardens let me know if we should keep this one

**Pull request merged**